cdap-app-templates/cdap-program-report/cdap-program-report-base/src/main/java/co/cask/cdap/report/ReportGenerationSpark.java

```diff
@@ -123,15 +144,36 @@ public void initialize(SparkHttpServiceContext context) throws Exception {
       readLimit = readLimitString == null ? 10000 : Integer.valueOf(readLimitString);
     }
 
+    /**
+     * Read the secure key file to read the security key bytes and initialize Cipher using AES and the mode
+     * encryption/decryption is based on the parameter cipherMode.
+     *
+     * @param cipherMode either encrypt or decrypt mode
+     * @return Initialized Cipher
+     * @throws IOException If the key file doesn't exist or if there are any exception while reading the file
```

Comment:
this doesn't match with the method signature.

Reply:
updated it.